nasa-gibs/worldview
# WV-2545: Uniform styling for tooltip components

## Description

This provides uniform styling for our Reactstrap uncontrolled tooltips and tooltips that includes extra padding and slight border radius.

## How To Test

1. `git checkout wv-2545`
2. `npm ci`
3. `npm run watch`
4. Check out the tooltips.
